code/modules/virus2/effect/stage_3.dm

```diff
@@ -530,3 +530,58 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/mob)
 	to_chat(mob, "<span class='notice'>You feel colorful!</span>")
 	mob.reagents.add_reagent(COLORFUL_REAGENT, 5)
 	mob.reagents.add_reagent(PAISMOKE, 5)
+
+/datum/disease2/effect/chimera
+	name = "Chimeral Xenosis"
+	desc = "Causes the infected's body to gradually mutate into a chimera of different alien species."
+	stage = 3
+	badness = EFFECT_DANGER_HARMFUL
+	chance = 1
+
+/datum/disease2/effect/chimera/activate(var/mob/living/mob)
+	if(!ishuman(mob))
+		return
+	var/mob/living/carbon/human/H = mob
+	var/list/valid_species = (all_species - list("Krampus", "Horror", H.species.name))
+
+	if(prob(80)) //most of the time we'll replace limbs
+		var/list/valid_organs = new()
+		for(var/datum/organ/external/E in H.organs)
+			if(!E.species || E.species == H.species)
+				valid_organs |= E
+		if(!valid_organs.len)
+			return //all our organs are already replaced
+		var/datum/organ/external/E = pick(valid_organs)
+		E.species = all_species[pick(valid_species)]
+		H.update_body()
+		//to_chat(mob, "<span class='notice'>Your [E.display_name] feels foreign.</span>")
+	else //the rest of the time we replace internal organs
+		var/list/valid_organs = new()
+		for(var/datum/organ/internal/I in H.internal_organs)
+			if(I.name != "brain" && !I.robotic && ispath(H.species.has_organ[I.name], I))
+				valid_organs |= I
+		if(!valid_organs.len)
+			return //all our organs are already replaced
+
+		var/datum/organ/internal/old_organ = pick(valid_organs)
+		var/list/valid_replacement_organs = new()
+		for(var/I_type in existing_typesof(/datum/organ/internal))
+			var/datum/organ/internal/I = new I_type()
+			if(istype(I, old_organ) && I.type != old_organ.type && !I.robotic)
```

Comment:
if I'm reading this correctly this will only replace human organs with non-human organs, and never the other way around (or between non-human species at all) because `istype(vox_eyes, human_eyes) == TRUE` but `istype(human_eyes, vox_eyes) == FALSE` and `istype(grey_eyes, vox_eyes) == FALSE`